addons/common/fnc_randPos.sqf

```diff
@@ -27,7 +27,7 @@ params [
     ["_radius", 0, [0]]
 ];
 
-private _position = _entity call CBA_fnc_getPos;
+private _position = +(_entity call CBA_fnc_getPos); // copy so we don't modify the input array
 
 _position set [0, (_position select 0) + (_radius - 2 * random _radius)];
 _position set [1, (_position select 1) + (_radius - 2 * random _radius)];
```

Comment:
To get a uniform distribution:
`_position getPos [_radius * sqrt (random 1), random 360]`
See http://mathworld.wolfram.com/DiskPointPicking.html